app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.html

```diff
@@ -1,7 +1,7 @@
 @(params: Map[String, Any])
-@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "This is Fulfilment House Due Diligence Scheme Beta submission email."){
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "Fulfilment House Due Diligence Scheme Beta"){
 <p style="margin: 0 0 30px; font-size: 19px;">
-    <p style="margin: 0 0 30px; font-size: 19px;">Hello Customer</p>
+    <p style="margin: 0 0 30px; font-size: 19px;">Hello</p>
```

Comment:
This is gonna be a spam flag and not good enough for a salutations, use their name.